### ISSUE TITLE
[QUESTION] How to disable whichwarp?

### ISSUE THREAD

**Issue by virgil710** [mawww/kakoune#4969]
### Question

Hello,guys!
When I move the cursor to the end of the line and press the right button, it will move to the next line. This function is called "**whichwrap**" in **vim**.
In kakoune, the function is enabled by default, I don't like this, I want to disable the function.
Any ideas?

**Comment by krobelus** [comment]
here is one way to do it for `l` and `L`. YMMV if you need this behavior on other keys

```kak
define-command -override my-no-wrap -params 1 %{
	try %{
		execute-keys -draft <semicolon><a-K>$<ret>
		execute-keys %arg{1}
	} catch %{
	}
}
map global normal l %{:my-no-wrap l<ret>}
map global normal L %{:my-no-wrap L<ret>}
```

**Comment by milkbrother666** [comment]
> here is one way to do it for `l` and `L`. YMMV if you need this behavior on other keys
> 
> ```kakscript
> define-command -override my-no-wrap -params 1 %{
> 	try %{
> 		execute-keys -draft <semicolon><a-K>$<ret>
> 		execute-keys %arg{1}
> 	} catch %{
> 	}
> }
> map global normal l %{:my-no-wrap l<ret>}
> map global normal L %{:my-no-wrap L<ret>}
> ```

Could you tell me how to use it ? I just started using Kakoune. Thank you!

**Comment by Screwtapello** [comment]
If it doesn't work, do any error messages show up in the `*debug*` buffer? (You can switch to it with `:buffer *debug*` inside Kakoune)

**Comment by milkbrother666** [comment]
> If it doesn't work, do any error messages show up in the `*debug*` buffer? (You can switch to it with `:buffer *debug*` inside Kakoune)

No, there are no any error messages, only one default line of prompts.

**Comment by krobelus** [comment]
try adding statements like `echo -debug here` inside the  command to see if it even gets there (the output  will show up in the `*debug*` buffer).
If not, try running  `:my-no-wrap l` (by literally typing that, including the colon).
You can use other keys like `<right>` instead of `l`, that should work too

**Comment by milkbrother666** [comment]
Thank you guys.
I've tried all the methods you guys have mentioned, but none of them have worked so far, so I'm thinking that maybe I should try to change my habits.  : - D
